pyproject.toml

```diff
@@ -71,18 +71,17 @@ botocore-dev = [
     # "colorama>0.3.0", # Windows requirement
 ]
 dev = [
-    "dill ~=0.3.3", # Used in test_patches.py
-    "docker ~=7.1",
-    "docutils >=0.16,<0.22", # Used in test_version.py
-    "moto[server,s3,sqs,awslambda,dynamodb,cloudformation,sns,batch,ec2,rds] ~=4.2.9",
-    "packaging ~=24.1", # Used in test_version.py
+    "dill ~= 0.3.3", # Used in test_patches.py
```

Comment:
@webknjaz had suggested to adopt the sparse notation. Why revert this?

Reply:
I just added whitespace to make it more readable, what do you mean by sparse notation, never heard of it.  PEP 440 suggests having spaces as well: https://peps.python.org/pep-0440/#version-specifiers

Reply:
I don't care either way, just want to make sure we don't convert back and forth all the time, depending on who authors or does reviews.

Reply:
I care in that I want it readable :)